protos/feast/core/DataSource.proto

```diff
@@ -146,6 +146,9 @@ message DataSource {
 
     // Watermark delay threshold for stream data
     google.protobuf.Duration watermark_delay_threshold = 4;
+
+    // kafka_settings field
+    StreamFormat kafka_settings = 5;
```

Comment:
is this necessary? In line 145 there's already a StreamFormat called `message_format`

Reply:
removed the settings